src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -188,13 +189,15 @@ def discrim(pols) -> tuple:
         sage: from sage.schemes.curves.zariski_vankampen import discrim
         sage: R.<x, y> = QQ[]
         sage: flist = (y^3 + x^3 - 1, 2 * x + y)
-        sage: discrim(flist)
-        (1,
-        -0.500000000000000? - 0.866025403784439?*I,
-        -0.500000000000000? + 0.866025403784439?*I,
-        -0.522757958574711?,
-        0.2613789792873551? - 0.4527216721561923?*I,
-        0.2613789792873551? + 0.4527216721561923?*I)
+        sage: L = list(discrim(flist))
+        sage: L.sort()
+        sage: L
```

Comment:
```suggestion
        sage: sorted((discrim(flist)))
```
would be more compact. Although is this output potentially given in a random order?